fluss-flink/fluss-flink-common/src/test/java/com/alibaba/fluss/flink/source/deserializer/FlussDeserializationSchemaTest.java

```diff
@@ -111,31 +125,92 @@ public void testGetProducedType() {
+        row.setField(4, "test bytes".getBytes(StandardCharsets.UTF_8));
+        row.setField(5, Decimal.fromBigDecimal(BigDecimal.valueOf(1_000_000L), 1, 1));
+        row.setField(6, (byte) 1);
+        row.setField(7, (short) 64);
+        row.setField(8, 1024);
+        row.setField(9, 2048L);
+        row.setField(10, 1.1f);
+        row.setField(11, 3.14d);
+        row.setField(12, (int) LocalDate.of(2025, 4, 21).toEpochDay());
+        row.setField(13, 36000);
+        row.setField(14, TimestampNtz.fromMillis(testTimestampInSeconds * 1000));
+        row.setField(15, TimestampLtz.fromEpochMillis(testTimestampInSeconds * 1000));
         ScanRecord scanRecord = new ScanRecord(row);
 
         // Create deserializer
         JsonStringDeserializationSchema deserializer = new JsonStringDeserializationSchema();
         // Test deserialization
-        deserializer.open(null);
+        deserializer.open(
+                new FlussDeserializationSchema.InitializationContext() {
+                    @Override
+                    public MetricGroup getMetricGroup() {
+                        return null;
+                    }
+
+                    @Override
+                    public UserCodeClassLoader getUserCodeClassLoader() {
+                        return null;
+                    }
+
+                    @Override
+                    public RowType getRowSchema() {
+                        return rowType;
+                    }
+                });
         String result = deserializer.deserialize(scanRecord);
 
+        String rowJson =
+                "{\"char\":\"a\",\"string\":\"test value\",\"boolean\":false,\"binary\":\"dGVzdCBiaW5hcnk=\",\"bytes\":\"dGVzdCBieXRlcw==\",\"tinyInt\":1,\"smallInt\":64,\"integer\":1024,\"bigInt\":2048,\"float\":1.1,\"double\":3.14,\"date\":\"2025-04-21\",\"timeWithoutTimeZone\":\"10:00:00\",\"timestampWithoutTimeZone\":\"2025-04-21T10:00:00\",\"timestampWithLocalTimeZone\":\"2025-04-21T10:00:00Z\"}";
+
         // Verify result
         assertThat(result).isNotNull();
         assertThat(result)
                 .isEqualTo(
-                        "{\"offset\":-1,\"timestamp\":-1,\"change_type\":\"INSERT\",\"row\":\"(42,test value)\"}");
+                        "{\"offset\":-1,\"timestamp\":-1,\"change_type\":\"INSERT\",\"row\":"
+                                + rowJson
+                                + "}");
 
         // Verify with offset and timestamp
         ScanRecord scanRecord2 = new ScanRecord(1001, 1743261788400L, ChangeType.DELETE, row);
         String result2 = deserializer.deserialize(scanRecord2);
         assertThat(result2).isNotNull();
         assertThat(result2)
                 .isEqualTo(
-                        "{\"offset\":1001,\"timestamp\":1743261788400,\"change_type\":\"DELETE\",\"row\":\"(42,test value)\"}");
+                        "{\"offset\":1001,\"timestamp\":1743261788400,\"change_type\":\"DELETE\",\"row\":"
+                                + rowJson
```

Comment:
1. add tests for null fields
2. change some fields and verify the result again. (as we are reusing the `ObjectNode` in `JsonStringDeserializationSchema`.)